src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -1060,6 +1060,13 @@ bool isLongLivedCaptureReferenceSyntax(SyntaxNode captureReferenceSyntax)
                                     break;
                             }
 
+                            if (syntax.Parent is CSharp.Syntax.WithExpressionSyntax withExpr
+                                && withExpr.Initializer.Expressions.Any()
+                                && withExpr.Expression == (object)syntax)
+                            {
+                                return true;
```

Comment:
>return true; [](http://example.com/codeflow?start=32&length=12)

> This is for scenarios like WithExprOnStruct_ControlFlow_WithCoalescingExpressionForValue so that the verifier won't complain about the long-lived capture with "Operation [{operationIndex}] in [{getBlockId(block)}] uses capture [{id.Value}] from another region. Should the regions be merged?" assertion above.

This would be more informative if the memo actually included the actual ids so that we could look at the graph and confirm that we indeed want to suppress the assertion.